compiler/src/dotty/tools/dotc/transform/Mixin.scala

```diff
@@ -214,6 +214,7 @@ class Mixin extends MiniPhase with SymTransformer { thisPhase =>
                 initFlags = stat.symbol.flags | PrivateLocal
               ).installAfter(thisPhase)
               stat.symbol.enteredAfter(thisPhase)
+            case _ =>
```

Comment:
What kind of non-ValDef statement are we getting here?

Reply:
The call to `Invoker.invoked` that is inserted when the template's parents are instrumented. Something like `Apply(Select(...), List(coverageId, coverageDir))`